src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj

```diff
@@ -10,4 +17,14 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="..\..\LICENSE" Pack="true" PackagePath="" />
+    <None Include="..\..\NOTICE" Pack="true" PackagePath="" />
+    <None Include="..\..\THIRD_PARTY_LICENSES" Pack="true" PackagePath="" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="..\..\icon.png" Pack="true" PackagePath="" />
+  </ItemGroup>
+
```

Comment:
I think we are missing settings for getting the assembly strong named using are snk file. Take a look at the other repos to see what we are missing and what our build pipeline is doing for doing strong naming.

This is required for the Visual Studio toolkit to be able to use the library.

Reply:
Fixed